utils/frame/benchmarking-cli/src/storage/weights.hbs

```diff
@@ -0,0 +1,106 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION {{version}}
+//! DATE: {{date}}
+//!
+//! DATABASE: `{{db_name}}`, RUNTIME: `{{runtime_name}}`
+//! SKIP-WRITE: `{{params.skip_write}}`, SKIP-READ: `{{params.skip_read}}`, WARMUPS: `{{params.warmups}}`
+//! STATE-VERSION: `V{{params.state_version}}`, STATE-CACHE-SIZE: `{{params.state_cache_size}}`
+//! WEIGHT-PATH: `{{params.weight_path}}`
+
+// Executed Command:
+{{#each args as |arg|}}
+// {{arg}}
+{{/each}}
+
+/// Storage DB weights for the {{runtime_name}} runtime and {{db_name}}.
+pub mod constants {
+	use frame_support::{parameter_types, weights::{RuntimeDbWeight, constants}};
+
+	parameter_types! {
+		{{#if (eq db_name "ParityDb")}}
+		/// ParityDB can be enabled with a feature flag, but is still experimental. These weights
+		/// are available for brave runtime engineers who may want to try this out as default.
+		{{else}}
+		/// By default, Substrate uses RocksDB, so this will be the weight used throughout
+		/// the runtime.
+		{{/if}}
+		pub const {{db_name}}Weight: RuntimeDbWeight = RuntimeDbWeight {
+			/// Time to read one storage item.
+			///
+			/// Stats [ns]:
+			///   Min, Max: {{underscore read.0.min}}, {{underscore read.0.max}}
+			///   Average:  {{underscore read.0.avg}}
+			///   Median:   {{underscore read.0.median}}
+			///   Mode:     {{underscore read.0.mode}}
+			///   StdDev:   {{read.0.stddev}}
+			///
+			/// Percentiles [ns]:
+			///   99th: {{underscore read.0.p99}}
+			///   95th: {{underscore read.0.p95}}
+			///   75th: {{underscore read.0.p75}}
+			read: 0 * constants::WEIGHT_PER_NANOS,
```

Comment:
The value here and the one below is up for discussion in chat.

Reply:
we should have `read.0.output_value` here, and let CLI choice pick the output value which is populated

Reply:
CLI should have all of these options above as options to populate with

Reply:
I haven't read the surrounding code much so maybe a dq, but why multiply by 0? Is 0 just a place holder?

Reply:
Added:  
- `--metric [ max | average | mean | p99 | p95 | p75]`. Default `average`
- `--mul <f64>` multiply the weight with a factor. Default `1.0`
- `--add <u64>` adds an offset to the final weight. Default `0`